spring-cloud-netflix-eureka-client-tls-tests/src/test/java/org/springframework/cloud/netflix/eureka/RestTemplateEurekaClientTest.java

```diff
@@ -82,6 +83,15 @@ public RestTemplateDiscoveryClientOptionalArgs forceRestTemplateDiscoveryClientO
 					eurekaClientHttpRequestFactorySupplier, new RestTemplateBuilderObjectProvider());
 		}
 
+		// Want to force reusing exactly the same bean as on production without excluding
```

Comment:
I know this class was already there, but as you're working on these changes, can you also rename this class  to `RestTemplateEurekaClientTests` and `BaseCertTest` to `BaseCertTests`.

Reply:
This appears to have already changed.

See https://github.com/spring-cloud/spring-cloud-netflix/commit/b23c5873192aebaf147f0566ca4817e4ed475d24